src/core/stdc/assert_.d

```diff
@@ -0,0 +1,52 @@
+/**
+ * D header file for C99.
+ *
+ * $(C_HEADER_DESCRIPTION pubs.opengroup.org/onlinepubs/009695399/basedefs/_assert.h.html, _assert.h)
+ *
+ * License: Distributed under the
+ *      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+ *    (See accompanying file LICENSE)
+ * Source:    $(DRUNTIMESRC core/stdc/_assert_.d)
+ * Standards: ISO/IEC 9899:1999 (E)
+ */
+
+module core.stdc.assert_;
+
+extern (C):
+@trusted:
+nothrow:
+@nogc:
+
+/****************************
+ * These are the various functions called by the assert() macro.
+ * They are all noreturn functions, although D doesn't have a specific attribute for that.
+ */
+
+version (CRuntime_DigitalMars)
+{
+    void _assert(const(void)* exp, const(void)* file, uint line);
+}
+else version (CRuntime_Microsoft)
+{
+    void _assert(const(char)* exp, const(char)* file, uint line);
```

Comment:
I see. A comment would be nice for the next time I stumble over this.

Should we add the _wassert version, too, similar to the various declaration on the other platforms?